src/communication/global_policy.hpp

```diff
@@ -1,21 +1,37 @@
 #pragma once
 
-#ifdef NMC_HAVE_MPI
-    #include "communication/mpi_global_policy.hpp"
+#include <string>
+
+namespace nest { namespace mc { namespace communication {
+    enum class global_policy_kind {serial, mpi, dryrun};
```

Comment:
I'd probably go here with a parallel: `auto global_policy_kind {"serial", "mpi", "dryrun"}`
and then make the `to_string` just an array deref.